SAMS api health check failing

Run bun scripts/check-sams-bugs.ts > bug-results.json
/home/runner/work/_temp/7e285228-e05a-4488-9bb2-49df72de15dc.sh: line 1: bun: command not found
Error: Process completed with exit code 127.

--- a/.github/workflows/sams-health-check.yml
+++ b/.github/workflows/sams-health-check.yml
@@ -25,6 +25,8 @@ jobs:
           node-version: "25"
           cache: true
 
+      - uses: oven-sh/setup-bun@v2
+
       - name: Install dependencies
         run: vp install
 
@@ -68,6 +70,8 @@ jobs:
         with:
           node-version: "25"
           cache: true
+
+      - uses: oven-sh/setup-bun@v2
 
       - name: Install dependencies
         run: vp install
@@ -177,7 +181,7 @@ jobs:
   # ── Job 4: Open a single GitHub issue summarising anything actionable ────────
   notify:
     name: Create Issue if Actionable
-    needs: [swagger-drift, bug-check, regenerate]
+    needs: [ swagger-drift, bug-check, regenerate ]
     if: always()
     runs-on: ubuntu-latest
     steps:
@@ -187,6 +191,8 @@ jobs:
         with:
           node-version: "25"
           cache: true
+
+      - uses: oven-sh/setup-bun@v2
 
       - name: Install dependencies
         run: vp install

--- a/.github/workflows/sams-health-check.yml
+++ b/.github/workflows/sams-health-check.yml
@@ -25,8 +25,6 @@ jobs:
           node-version: "25"
           cache: true
 
-      - uses: oven-sh/setup-bun@v2
-
       - name: Install dependencies
         run: vp install
 

--- a/scripts/check-sams-bugs.ts
+++ b/scripts/check-sams-bugs.ts
@@ -10,6 +10,9 @@
  * Bug reference: .agents/skills/sams-api/BUGS.md
  */
 
+import "varlock/auto-load";
+import { ENV } from "varlock/env";
+
 const BASE_URL = "https://www.volleyball-baden.de/api/v2";
 
 // Known UUIDs (season 2025/26) — see .agents/skills/sams-api/API-OVERVIEW.md
@@ -283,7 +286,7 @@ async function checkBug8(apiKey: string): Promise<BugResult> {
 }
 
 async function main(): Promise<void> {
-	const apiKey = process.env.SAMS_API_KEY;
+	const apiKey = ENV.SAMS_API_KEY;
 	if (!apiKey) {
 		console.error("Error: SAMS_API_KEY is not set.");
 		process.exit(1);

